app/views/tasks/_new_tasks.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="new_tasks">
+  <table class="table table-bordered">
+    <thead>
+    <tr>
+      <td>New</td>
+    </tr>
+    </thead>
+    <tbody>
+    <% @tasks_not_started.each do |task| %>
```

Comment:
the same as [above](https://github.com/dKotetunov/TasksManager/pull/2/files#r27945248)